File: programmers/js/level00/120891.js
// 369게임
function solution(order) {
  return String(order)
    .split('')
    .filter(num => num === '3' || num === '6' || num === '9')
    .length;
}

// 다른 풀이
// function solution(order) {
//   var answer = [...order.toString().matchAll(/[3|6|9]/g)].length;
//   return answer;
// }

File: programmers/java/level00/120891.java
// 369게임

import java.util.Arrays;

class Solution {
    public int solution(int order) {
        return (int) Arrays.stream(String.valueOf(order).split(""))
            .filter(num -> num.equals("3") || 
                           num.equals("6") || 
                           num.equals("9"))
            .count();
    }
}